pyproject.toml

```diff
@@ -28,38 +28,41 @@ classifiers = [
     "Operating System :: OS Independent",
 ]
 dependencies = [
-        "colored",
-        "tqdm",
-        "pydicom",
-        "click",
-        "python-box<7.0.0",
-        "numpy",
-        "SimpleITK",
-        "matplotlib",
-        "python-Levenshtein",
-        "bs4",
-        "selenium",
-        "radioactivedecay",
-        "questionary",
-        "pandas",
-        "openpyxl",
+    "colored",
+    "tqdm",
+    "pydicom",
+    "click",
+    "python-box<7.0.0",
+    "numpy",
+    "SimpleITK",
+    "matplotlib",
+    "python-Levenshtein",
+    "bs4",
+    "selenium",
+    "radioactivedecay",
+    "questionary",
+    "pandas",
+    "TotalSegmentator",
```

Comment:
Does TotalSegmentator mandatory
Because it's quite complicated to install it with cuda gpu

Reply:
yes, we can make it optional